server/src/main/scala/com/cloudera/livy/server/interactive/InteractiveSession.scala

```diff
@@ -215,24 +215,31 @@ class InteractiveSession(
     stop()
   }
 
-  def executeStatement(content: ExecuteRequest): Statement = {
+  def executeStatement(content: ExecuteRequest): Statement = synchronized {
```

Comment:
Just to make sure statementId is unique in one session, so I add synchronized here.